### PR TITLE
adds rainbow footer

### PR DESCRIPTION
**Pull request by georgeweiler**
Screen Shot 2020-10-20 at 4 23 10 PM
Screen Shot 2020-10-20 at 4 23 31 PM

### CODE REVIEW

**Comment by georgeweiler** [comment]
@CarlBeek @hwwhww Last I heard we are still waiting for legal copy confirmation on this?

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+          <Link external to="https://ethereum.org/en/eth2/#roadmap">
+            Launch Phases
+          </Link>
+          <Link
+            external
+            to="https://docs.google.com/spreadsheets/d/15tmPOvOgi3wKxJw7KQJKoUe-uonbYR6HF7u83LR5Mj4/edit#gid=842896204"
+          >
+            Economics
+          </Link>
+          <Link external to="http://activate.codefi.network/eth2">
```
Comment (georgeweiler) [inline src/components/Footer.tsx:83]: something to note: this page will not go live until the EF mainnet launch

**Comment by hwwhww** [comment]
@georgeweiler RWD design looks good! 👍 

We are still waiting for the policy page content. *If* we need links for multiple static pages (e.g., "Privacy policy", "Terms of Use"), can we replace the "Ethereum 2 Launchpad" column with new pages?

By the way, where does "A Contribution 'originates' from a designated place, then offering..." come from? Is it a stub?

**Review by Battenfield — CHANGES_REQUESTED** [review]
Let's repleace the previous dark blue footer on the homepage for the new 🌈

**Comment by hwwhww** [comment]
It seems this PR is replaced by #231 ?

**Comment by georgeweiler** [comment]
closing, because this is replaced by #231

**Pull request closed without merging**